sync/class.jetpack-sync-listener.php

```diff
@@ -268,6 +268,7 @@ function get_actor( $current_filter, $args ) {
 			'is_wp_rest'       => defined( 'REST_REQUEST' ) ? REST_REQUEST : false,
 			'is_ajax'          => defined( 'DOING_AJAX' ) ? DOING_AJAX : false,
 			'is_wp_admin'      => is_admin(),
+			'is_cli'           => defined( 'WP_CLI' ) ? WP_CLI : false,
```

Comment:
As a note, if we used something like `Jetpack_Constants::defined( 'WP_CLI' ) && Jetpack_Constants::get_constant( 'WP_CLI' )` here, then you could change the value in the PHPUnit tests with something like `Jetpack_Constants::set_constant( true )` and verify that it gets picked up correctly.